docs/content/preview/develop/common-patterns/timeseries/global-ordering.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Global Ordering By Time
+headerTitle: Global Ordering By Time
+linkTitle: Global ordering by time
+description: Distribute your time-ordered data and retrieve fast
+headcontent: Distribute your time-ordered data and retrieve fast
+menu:
+  preview:
+    identifier: timeseries-global-ordering
+    parent: common-patterns-timeseries
+    weight: 100
+type: docs
+---
+
+Here, we will see how to distribute your entire time-ordered dataset across tablets and retrieve data efficiently.
+
+## Setup
+
+{{<cluster-setup-tabs>}}
+
+## Distributing on timestamp
+
+Let us consider a speed metrics tracking system that tracks the data from the speed sensor of many cars. A simple schema would be,
+
+```sql
+CREATE TABLE global_order1 (
+    ts timestamp, /* time at which the event was generated */
+    car varchar, /* name of the car */
+    speed int, /* speed of your car */
+    PRIMARY KEY(ts ASC)
+);
+```
+
+This would store the speed data points of different cars as they arrive in the system.
+
+Let us insert some sample data into the table.
+
+```sql
+INSERT INTO global_order1 (ts, car, speed) 
+        (SELECT '2023-07-01 00:00:00'::timestamp + make_interval(secs=>id),
+            'car-' || ceil(random()*2), ceil(random()*60)
+            FROM generate_series(1,100) AS id);
+```
+
+Now let us retrieve some data from the table.
+
+```sql
+SELECT * FROM global_order1;
+```
+
+```output
+         ts          |  car  | speed
+---------------------+-------+-------
+ 2023-07-01 00:00:01 | car-1 |    50
+ 2023-07-01 00:00:02 | car-2 |    25
+ 2023-07-01 00:00:03 | car-1 |    39
+ 2023-07-01 00:00:04 | car-1 |    49
+ 2023-07-01 00:00:05 | car-2 |     3
+ 2023-07-01 00:00:06 | car-2 |    22
+ 2023-07-01 00:00:07 | car-2 |    25
+ 2023-07-01 00:00:08 | car-1 |    58
+ 2023-07-01 00:00:09 | car-2 |    55
+```
+
+You will notice that the data is automatically ordered on time. This is because the table is set to be sorted on the `ts` by `PRIMARY KEY(ts ASC)`. This ensures that the data is sorted and all closeby data resides in the same tablet. This sort of order makes it efficient for range queries to get all data within a specific time range. For example,
+
+```sql
+SELECT * FROM global_order1 WHERE ts > '2023-07-01 00:01:00' AND ts < '2023-07-01 00:01:05';
+```
+
+```output
+         ts          |  car  | speed
+---------------------+-------+-------
+ 2023-07-01 00:01:01 | car-1 |    21
+ 2023-07-01 00:01:02 | car-2 |    58
+ 2023-07-01 00:01:03 | car-1 |    57
+ 2023-07-01 00:01:04 | car-2 |    60
+```
+
+But as the data grows, the tablet splits and one-half moves to a different tablet ensuring scalability and so on. This also means that the grows in one shard before moving to the next tablet. Also because a specific range could be in a single shard, this could lead to Let us see how to distribute the ordered data across different tablets.
```

Comment:
"Also because a specific range could be in a single shard, this could lead to... "
lead to what ?

Reply:
fixed.